src/lib/media/picker.shared.ts

```diff
@@ -17,14 +17,14 @@ export async function openPicker(opts?: ImagePickerOptions) {
   })
 
   if (response.assets && response.assets.length > 4) {
-    Toast.show('You may only select up to 4 images')
+    Toast.show('You may only select up to 4 images', 'exclamation-circle')
```

Comment:
While you're updating toasts, would it be possible to mark this for localization please? 🙏